terminal_table/terminal_table.py

```diff
@@ -137,7 +137,8 @@ def get_wrapped_row(row, max_column_widths):
 
     @staticmethod
     def get_row_height(wrapped_row):
-        return max(len(col) for col in wrapped_row)
+        max_heights = [len(col) for col in wrapped_row]
```

Comment:
this crashes on empty list, i wondered if it should return 0 on empty list? I can undo if you don't agree